state-chain/traits/src/lib.rs

```diff
@@ -724,6 +724,7 @@ pub trait DepositApi<C: Chain> {
 		lp_account: Self::AccountId,
 		source_asset: C::ChainAsset,
 		boost_fee: BasisPoints,
+		refund_address: Option<ForeignChainAddress>,
```

Comment:
This should not be optional. 

If the LP hasn't registered a refund address, they can't open a channel, so we should always have this.